test/model/src/core/verify.cpp

```diff
@@ -177,6 +175,16 @@ kv_table_verifier::verify(WT_CONNECTION *connection, kv_checkpoint_ptr ckpt)
         /* The model has more key-value pairs than the database. */
         if (model_cursor.has_next())
             throw verify_exception("There are still more key-value pairs in the model.");
+
+        /*
+         * Only do the verify if all the tests have passed. Make sure all cursors are closed before
+         * calling verify to avoid EBUSY. We don't need to tolerate non-fatal errors for the final
+         * step.
+         */
+        ret = session->verify(session, uri.c_str(), "strict");
```

Comment:
As suggested in the ticket, we should check if we are in attached storage or disagg mode here.
I don't think we have access to any info on disagg at this point. Could be a field for that class or we could check if the URI is layered?